src/main/java/com/dataart/demo/hello.java

```diff
@@ -41,7 +41,7 @@ public static void main(String[] args) throws Exception {
 	static int cnt=0;
 	
 	public String getMessage() {
-		return String.format("Hello, HTTP! Count=%d\n",cnt);
+		return String.format("Hello, HTTP! Count=%d\n",cnt+1);
```

Comment:
your code is wrong here!!

Reply:
What is wrong there? 
Why do you  you suggest cnt+1 instead of cnt?

PS Anyway, you are welcome to try and see if it makes any difference :)